learning_resources_search/serializers.py

```diff
@@ -287,16 +288,41 @@ class ContentFileSearchRequestSerializer(SearchRequestSerializer):
     )
 
 
+class AggregationValue(TypedDict):
+    key: str
+    doc_count: int
+
+
 def _transform_aggregations(aggregations):
-    formatted_aggregations = {}
-    for key, value in aggregations.items():
-        if value.get("buckets"):
-            formatted_aggregations[key] = value.get("buckets")
-        elif value.get(key, {}).get("buckets"):
-            formatted_aggregations[key] = value.get(key, {}).get("buckets")
-        else:
-            formatted_aggregations[key] = value.get(key, {}).get(key, {}).get("buckets")
-    return formatted_aggregations
+    def get_buckets(key, aggregation):
+        if "buckets" in aggregation:
```

Comment:
In the old version of `_transform_aggregations`, if `"buckets"` key existed but its value was an empty list, the if/else would fall through to old-line-298. This resulted in `null` values for aggregations in some casts. (Example: course_feature, https://mitopen-rc.odl.mit.edu/api/v1/learning_resources_search/?resource_type=podcast&aggregations=certification,professional,course_feature,level,topic,platform,offered_by&q=BroadIgnite)